app/package.json

```diff
@@ -15,7 +15,9 @@
   },
   "main": "build/main/start.cjs.js",
   "scripts": {
-    "build": "node scripts/build.js",
+    "build": "cross-env MODE=production npm-run-all build:*",
```

Comment:
could we use prebuild instead of npm-run-all here? seems more natural to mee but I dont have real arguments